controllers/accounts.go

```diff
@@ -17,29 +17,29 @@ import (
 	"github.com/google/uuid"
 )
 
-type accountsAPI struct {
+type AccountsAPI struct {
 	accountsv1.UnimplementedAccountsAPIServer
 
-	auth   auth.Service
-	logger *zap.Logger
-	repo   models.AccountsRepository
+	Auth   auth.Service
+	Logger *zap.Logger
+	Repo   models.AccountsRepository
```

Comment:
Looks like you modified all `*API` structures to make all the variables within them public. Is there a specific reason for that? Do you need to access them from another package or else?

Reply:
It was from the tests package, but it's unnecessary